bedrock/firefox/templates/firefox/features/send-tabs.html

```diff
@@ -0,0 +1,151 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import google_play_button with context %}
+
+{% add_lang_files "firefox/shared" "firefox/sync" %}
```

Comment:
Definitely don't import a lang file for a shared string, we can copy over the translation